styles.css

```diff
@@ -1,6 +1,21 @@
-
 @import url('https://fonts.googleapis.com/css2?family=Poppins&family=Roboto&display=swap');
 
+body {
+  position: relative;
+  width: 340px;
+  height: 5410px;
+  background: #e5e5e5;
+  margin: 0;
+  padding: 0;
```

Comment:
- [ ] I can't pass without noticed the style you did, and it is :1st_place_medal: :1st_place_medal: :1st_place_medal: , however you defined a static  `height`, it's a good practice to make the `height` dynamic ever the one you use on your body. Kindly make sure you change that :100: